vertx-websocket-chat/README.adoc

```diff
@@ -0,0 +1,56 @@
+= Vertx-Websocket Chat: A Camel Quarkus example
+:cq-example-description: Demonstration of how to configure a WebSocket server and interact with connected peers.
```

Comment:
Seems the cq-maven-plugin wants to enforce a specific description style and checks for a particular [prefix](https://github.com/l2x6/cq-maven-plugin/blob/main/maven-plugin/src/main/java/org/l2x6/cq/maven/UpdateExamplesJsonMojo.java#L50). If you modify things like the following, it'll hopefully work ok.

```suggestion
:cq-example-description: An example that shows how to configure a WebSocket server and interact with connected peers.
```